src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  place-content: center;
+  grid-template-columns: 200px;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+@media screen and (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
```

Comment:
For the media query starting at 488px, it's a good practice to also define a `max-width` to ensure that this style only applies within a specific range. This can prevent conflicts with other media queries and make the breakpoints more predictable.